vendor/idrive_vnc.js

```diff
@@ -0,0 +1,44 @@
+/*
+// IDrive VNC Java Script file
+*/
+
+function handleConnect() {
+  var canvas = document.getElementsByTagName('canvas')[0];
+  canvas.focus();
+  canvas.style.paddingLeft = '10px';
```

Comment:
@pawan-idrive why is the styling applied here? Can this not go into the css?

Reply:
Will move it